pkg/appender/store.go

```
@@ -218,7 +221,8 @@ func (cs *chunkStore) chunkByTime(t int64) *attrAppender {
 		if err != nil {
 			return nil
 		}
-		cur.initialize(part.NextPart(t), t) // TODO: next part
+		nextPart, _  := part.NextPart(t)
```

Why error is ignored here?